src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -344,7 +344,7 @@ private static object GetTypedProperty(IPublishedContent content, Type type)
                                     if (toConvert != null)
                                     {
                                         var converter = DependencyResolver.Current.GetService(toConvert) as TypeConverter;
-                                        if (converter != null)
+                                        if (converter != null && converter.CanConvertFrom(propertyType))
```

Comment:
Question: What should we be passing to `CanConvertFrom`? `propertyType` or `propertyValue.GetType()`?

e.g. The intended type, or the type of the current value?